safehttp/incoming_request.go

```diff
@@ -14,9 +14,67 @@
 
 package safehttp
 
-import "net/http"
+import (
+	"errors"
+	"net/http"
+	"strings"
+)
 
 // IncomingRequest TODO
 type IncomingRequest struct {
 	req *http.Request
 }
+
+// QueryForm TODO
+func (r IncomingRequest) QueryForm() (*Form, error) {
+	if r.req.Method != "GET" {
+		return &Form{}, errors.New("got request method " + r.req.Method + ", want GET")
```

Comment:
Use `errors.New` only when the message is not parametrized. Use `fmt.Errorf` instead here.

Reply:
Resolved in [2dc01f6](url)